multigeiger/ble.cpp

```diff
@@ -0,0 +1,141 @@
+// Project: Simple Multi-Geiger
+// Modul: BLE transmission of CPM data
+//
+// code related to the transmission of the measured data via Bluetooth Low Energy
+// Uses GATT Heart Rate Measurement Service for notifications with CPM update
+//
+// Heart Rate Measurement = Radiation CPM, Energy Expense = Rolling Packet Counter
```

Comment:
the heart rate part is somehow duplicate.

Reply:
yeah, I kept the official names...